test/fixtures/v2/documentRopstenValidWithDIDSignedProofBlockProperHash.ts

```diff
@@ -0,0 +1,55 @@
+import { v2, WrappedDocumentWithProof } from "@govtechsg/open-attestation";
```

Comment:
Questions regarding the 3 test files added
1. how did you add the proof (manually ? ur own tool ? I think if we want this we need to add it into open-attestation)
1. Did you sign the whole content ?
1. Any reason you kept the certificate store ? (other than open-attestation require one but then it's not good and we need to update oa)

Reply:
1. Yes we signed using our own tooling then copied over.  Happy to pull that code into an appropriate spot in OA as you see fit.  Perhaps worth a chat about this.

2. You are right! We just signed the `targetHash` to reduce complexity of computing our own hash of the content.

3. Mainly so I could use the same `signature` block so the hash tests would pass during integration testing and to conform yes.  Also the certificate store can still be used for revocation registry in the short term if desired.